src/Validation/StarterKitValidation.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace AcquiaCMS\Cli\Validation;
+
+use AcquiaCMS\Cli\Exception\AcmsCliException;
+use JsonSchema\Validator;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Validation service to validate each key in starter kit.
+ */
+class StarterKitValidation {
+  /**
+   * Holds the symfony console output object.
+   *
+   * @var \Symfony\Component\Console\Output\OutputInterface
+   */
+  protected $output;
+
+  /**
+   * Constructs an object.
+   *
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *   Returns an absolute root path to project.
+   */
+  public function __construct(
+    OutputInterface $output) {
+    $this->output = $output;
+  }
+
+  /**
+   * Loop each starter-kit and send it for validation.
+   */
+  public function validateStarterKits(array $schema, array &$starterkits): void {
+    $errorMessage = '';
+    foreach ($starterkits as $name => $starterkit) {
+      $errorMessage .= $this->validateStarterKit($schema, $starterkit, $name);
+    }
+    if ($errorMessage) {
+      throw new AcmsCliException($errorMessage);
+    }
+  }
+
+  /**
+   * Validates starter-kit.
+   */
+  public function validateStarterKit(array $schema, array $starterkit, string $name): ?string {
```

Comment:
It seems like the return type ?string might not be necessary here. Please assign a suitable return type.
```suggestion
  public function validateStarterKit(array $schema, array $starterkit, string $name): string {
```

Reply:
this will return string type as we are passing error message in string format.

Reply:
done